docs/anchor-platform.md

```diff
@@ -74,8 +72,17 @@ docker build -t stellar-anchor-platform:latest .
 ```
 
 Docker Run:
+
+Note: secrets (credentials, tokens, etc...) are passed to the application via environment variables. Use `-e` to pass in
+each required environment variables ([Environment Variables](../platform/src/main/resources/example.env) )
 ```shell
-docker run -v {/local/path/to/config/file/}:/config -p 8081:8081 stellar-anchor-platform:latest --anchor-reference-server
+docker run -v {/local/path/to/config/file/}:/config -p 8081:8081 stellar-anchor-platform:latest --anchor-reference-server \
+-e JWT_SECRET=secret \
+-e SEP10_SIGNING_SEED=SAX3...C3AW5X \
+-e CIRCLE_API_KEY=QVBJX0...NjMyZTQ5NWJhNDdlZg== \
+-e PAYMENT_GATEWAY_STELLAR_SECRET_KEY=secret \
+-e POSTGRES_USERNAME=postgres \
+-e POSTGRES_PASSWORD=password
```

Comment:
Do you always need to use `-e`? Or can you rely solely on the values from the `.env` file?

Reply:
this will run in a docker container so you either have to explicitly pass in the env variables or pass in the .env file.

Reply:
i'll add an example of running the command with a .env file